composer.json

```
@@ -9,18 +9,20 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
-        "nyholm/psr7": "^1.3",
-        "php-http/httplug": "^2.0",
+        "php": "^8.1",
+        "nyholm/psr7": "^1.8.1",
+        "php-http/httplug": "^2.4",
         "psr/http-client": "^1.0",
+        "psr/http-message": "^1.0 || ^2.0",
         "symfony/options-resolver": "^2.6 || ^3.4 || ^4.4 || ^5.0 || ^6.0 || ^7.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.2 || ^3.0",
-        "php-http/client-integration-tests": "^3.0",
-        "php-http/message": "^1.9",
-        "php-http/client-common": "^2.3",
-        "phpunit/phpunit": "^8.5.23 || ~9.5"
+        "friendsofphp/php-cs-fixer": "^3.51",
+        "php-http/client-integration-tests": "3.0.1",
```

See the message of commit https://github.com/php-http/socket-client/pull/75/commits/61bb8f5fd07e64a6a7b921071545e07a6512b541

To rollback this - it needed to wait a release moment for https://github.com/php-http/client-integration-tests/pull/60 and update the minimal version (and drop an extra entry about `psr-http/message-factory`).

---

merged and tagged your cleanup: https://github.com/php-http/client-integration-tests/releases/tag/3.1.1

```suggestion
        "php-http/client-integration-tests": "^3.1.1",
```